tests/testsuite/cargo_alias_config.rs

```diff
@@ -73,6 +76,56 @@ fn recursive_alias() {
         .run();
 }
 
+#[cfg(unix)]
```

Comment:
cfg(unix) because I use a small bash program in the test.
Open to other ideas. Just needed a small program which echoed back out args.